### DISCUSSION TITLE
dti TensorModel fitting issue

### DISCUSSION THREAD

**Discussion by aga-sierhej** [dipy/dipy#2640] [Reconstruction]
## Description
Hi, 
I have an issue with fitting the diffusion tensor model using dipy_fit_dti.
I've been following the tutorial, however, the fitting takes too long (on another machine it takes around 1 min per subject, on my current machine it cannot perform a fitting for 60+ min). 
After the interruption, the code tends to be stuck in one particular place.
<img width="1220" alt="Screenshot 2022-06-28 at 16 50 54" src="https://user-images.githubusercontent.com/99027732/176227431-77c2d1bd-32c7-4a9a-82bc-7773a89ff8d1.png">

 I don't know what kind of issue is this. Any help would be appreciated.

## Way to reproduce
[If reporting a bug, please include the following important information:]
- [ ] Code example
- [ ] Relevant images (if any)
- [ ] Operating system and version: macOS-10.16-x86_64-i386-64bit
- [ ] Python version: Python 3.9.12
- [ ] dipy version: 1.5.0
- [ ] Visual Code Studio: Version: 1.68.1 (Universal)
- [ ] dependency version (numpy, scipy, nibabel, h5py, cvxpy, fury)
    * NumPy 1.21.5
    * SciPy 1.7.3
    * Nibabel 4.0.1
    * H5py 3.6.0
    * Cvxpy None
    * fury None

**Comment by skoudoro** [comment]
What is the difference between the 2 machines? Can you describe the specifications and operating system?
